xml/System.Text.Json.Serialization.Metadata/JsonTypeInfo.xml

```diff
@@ -255,9 +347,25 @@ This API is intended for use by the output of the System.Text.Json source genera
         <ReturnType>System.Collections.Generic.IList&lt;System.Text.Json.Serialization.Metadata.JsonPropertyInfo&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the list of <see cref="T:System.Text.Json.Serialization.Metadata.JsonPropertyInfo" /> metadata corresponding to the current type.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This property is only applicable to metadata of kind <xref:System.Text.Json.Serialization.Metadata.JsonTypeInfoKind.Object>.
+             For other kinds, an empty, read-only list will be returned.
+            
+The order of <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo> entries in the list determines the serialization order,
+             unless either of the entries specifies a non-zero <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.Order> value,
+             in which case the properties will be stable sorted by <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.Order>.
```

Comment:
@eiriktsarpalis @layomia there is no "Order" API in `JsonPropertyInfo` (at least not in Preview6, that's the branch I got this documentation from). Was it renamed? Is it a typo?

Reply:
It will be released with Preview 7.

Reply:
Oooooh it was internal in preview6: https://github.com/dotnet/runtime/blob/d3fa592f6d5d678edb8aea7568812b167c72e622/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs#L527-L530 

and made public in preview7: https://github.com/dotnet/runtime/blob/bc64ffee79272691471a07aa6855fcb892096bf2/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs#L632


cc @gewarren we are good here.

Reply:
Note: we are good here in the sense that we can keep the docs for now, and the warning will go away when we bring in prev7.